bpf/bpf_lxc.c

```diff
@@ -1035,10 +1035,16 @@ __section_tail(CILIUM_MAP_POLICY, LXC_ID) int handle_policy(struct __sk_buff *sk
 
 	ifindex = skb->cb[CB_IFINDEX];
 
-	if (ifindex)
-		return redirect(ifindex, 0);
-	else
+	if (ifindex) {
+		ret = redirect(ifindex, 0);
```

Comment:
Can we even get visibility at all?

Reply:
Either via perf drop monitor or bpf/kprobes.

Reply:
Ah, that's too bad. I might respin an extra debug log message for this case though, since in verbose mode it only prints the packet and there is no indication of what is happening - expected success to redirect or not.